packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2141,35 +2141,40 @@ function commitRootImpl(root, renderPriorityLevel) {
 }
 
 function commitBeforeMutationEffects(fiber: Fiber) {
```

Comment:
As a convention, I've used `firstChild` as the name of the parameter when it's conceptually passed a linked list of children rather than a specific fiber.

https://github.com/facebook/react/blob/master/packages/react-reconciler/src/ReactFiberBeginWork.new.js#L2453